### ISSUE TITLE
Nested switch statements with no break blend (Generic.WhiteSpace.ScopeIndent.IncorrectExact)

### ISSUE THREAD

**Issue by chrisgraham** [squizlabs/PHP_CodeSniffer#2738]
`Generic.WhiteSpace.ScopeIndent.IncorrectExact` is being triggered:

```
switch ($x) {
    case 'a':
        switch ($y) {
            case 'a1':
                break;
        }
        // no break

    case 'b':
        break;
}
```

It seems to be that the missing break statement makes the parser think the final case statement is a part of the inner switch block.
`// no break` is not needed other than to stop a different sniff (correctly) triggering.

**Comment by chrisgraham** [comment]
Ah sorry, I was using `phpcs` from Ubuntu, you've fixed it already.
